build/docs/classes/DocsBuilder.php

```diff
@@ -226,6 +244,31 @@ private function updateClients(array $services)
         }
     }
 
+    private function updateAliases(array $services, array $compatibleVersions)
```

Comment:
I think this approach is fine for now. Let's try to see if we can come up with a better approach going forward though.